src/Textfield.js

```diff
@@ -60,13 +60,32 @@ class Textfield extends React.Component {
         }
     }
 
+    labelElement(customId) {
+        const { label } = this.props;
+
+        if (typeof label === 'string') {
+            return (
+                <label className="mdl-textfield__label" htmlFor={customId}>
+                    {label}
+                </label>
+            );
+        } else if (typeof label === 'object') {
+            return label;
+        }
+
+        return null;
```

Comment:
Impossible to reach because `label` is required.

So you can just return `label` if it's not a string